hack/generate/olm_bindata.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+filename="olm_bindata-"$1.go
+echo $filename
+
+function get_olm_manifests() {
+    mkdir olm-manifests
+    echo "downloading olm manifests for version ${1}"
+    wget -O olm-manifests/olm.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/olm.yaml"
+    wget -O olm-manifests/crds.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/crds.yaml"
+}
+
+function remove_olm_manifests {
+    rm -rf olm-manifests
+}
+
+# check for files starting with the name "olm-bindata" inside internal/olm folder
+function delete_old_olmbindata {
+    echo "Deleting previous versions of olm-bindata files if they exist"
+    find internal/olmbindata -maxdepth 1 -type f -name olm_bindata-* -exec rm {} \;
+}
+
+FILE=internal/olm/$filename
```

Comment:
Why is this file in `internal/olm`? Shouldn't it be in `internal/olmbindata`?

Also I would prefer all bindata be kept in the same place, organized like
```console
$ tree internal/bindata
internal/bindata
├── olm
|   ├── versions.go
│   └── manifests-0.15.1.go
...
```
So the type of bindata is organized by name. `versions.go` would contain `HasVersion() bool`.

Reply:
Have named there package to be `olmbindata` so that it can be excluded from license header/error message checks.
```
internal/bindata
└── olmbindata
    ├── olm_bindata-0.15.1.go
    └── olm_versions.go
```